pkg/ui/markdown/parser.go

```diff
@@ -0,0 +1,55 @@
+package markdown
+
+import (
+	"strings"
+)
+
+// ParseMarkdownSections parses a markdown string and returns a map of section titles to their content
+func ParseMarkdownSections(content string) map[string]string {
```

Comment:
1. I think this was not required we can create `n` number of markdown and that should be fine.
2. This function was used in workflow but i would say it does not work with the current version of workflow.md. I know this because i wrote a short test case because i could not believe the function worked for `workflow.md`.